2015/IsCallable.js

```diff
@@ -1,5 +1,14 @@
 'use strict';
 
+var hasSymbols = require('has-symbols')();
+var callBound = require('../helpers/callBound.js');
+
+var $toStr = callBound('%Object.prototype.toString%');
+
 // http://www.ecma-international.org/ecma-262/5.1/#sec-9.11
 
-module.exports = require('is-callable');
+module.exports = function IsCallable(argument) {
+	// some older engines say that typeof /abc/ === 'function',
+	return typeof argument === 'function'
```

Comment:
this isn't actually useful though, because it will report `class` constructors as callable.

Reply:
Well, that’s what [`IsCallable`](https://tc39.es/ecma262/#sec-iscallable) says to do.